### PR TITLE
Copy updates

### PR DESCRIPTION
**Pull request by hannajones**
Took another look at the doc and only found one word change and an extra whitespace.

### CODE REVIEW

File: kitsune/sumo/static/sumo/js/form-wizard-sign-in-step.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export class SignInStep extends BaseFormStep {
           </p>
 
           <p class="for-sign-in">
-            ${gettext("A Firefox account allows you to securely sync your data when signing in from other devices and gives access to other fantastic Mozilla services.")}
+            ${gettext("A Firefox account allows you to securely sync your data when logging in from other devices and gives access to other fantastic Mozilla services.")}
```
Comment (mikeconley) [inline kitsune/sumo/static/sumo/js/form-wizard-sign-in-step.js:23]: Hey @hannajones - it turns out that we _did_ want this to be "signing", but the document was never updated to reflect that. I've corrected the document, and then I think we can revert this change.

**Review by mikeconley — APPROVED** [review]

File: kitsune/sumo/static/sumo/js/form-wizard-sign-in-step.js (hunk 1 above)
Comment (hannajones) [reply kitsune/sumo/static/sumo/js/form-wizard-sign-in-step.js]: thanks for clarifying! In that case I'll just commit the whitespace removal I guess 😂
